### DISCUSSION TITLE
IMPORTANT! Security Vulnerability Detected in your Repository (Issue #1)

### DISCUSSION THREAD

**Discussion by aalgohary** [community/community#139488] [Code Security]
### Select Topic Area

Show & Tell

### Body

<img width="1424" alt="image" src="https://github.com/user-attachments/assets/f619d410-1551-46a3-8848-8322146bcb9c">
I received this email and with some search found that it's a phishing email and wanted to let you know

**Comment by AESMatias** [comment]
I've never see something like that using GitHub like a hook, thank you for letting us know, definitively is a phishing attempt.

**Comment by ghostinhershell** [comment]
Hi @aalgohary, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
